src/audio/module_adapter/module_adapter_ipc4.c

```diff
@@ -100,6 +100,7 @@ int module_set_large_config(struct comp_dev *dev, uint32_t param_id, bool first_
 			    bool last_block, uint32_t data_offset_size, const char *data)
 {
 	struct processing_module *mod = comp_get_drvdata(dev);
+	const struct comp_driver *const drv = mod->dev->drv;
```

Comment:
wouldn't it be shorter to extract `ops = mod->dev->drv->adapter_ops` here and use `ops` everywhere  in all cases?